### PR TITLE
change return dicitonary for DataCollatorForNextSentencePrediction from masked_lm_labels to labels

### PR DESCRIPTION
**Pull request by gmihaila**
# What does this PR do?

The `masked_lm_labels` argument from  *DataCollatorForNextSentencePrediction*  is deprecated and will be removed in a future version, use `labels` instead. I changed the dictionary key from `masked_lm_labels ` to `labels`.

It will avoid any future errors when `masked_lm_labels` won't be used anymore.

Not a lot of people use the  *DataCollatorForNextSentencePrediction* and I think this will get overlooked in the future if not fixed.

It fixes warning that appear when using `trainer`

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dimiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests), 
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case. **Not the case**
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation). **Not needed.**
- [x] Did you write any new necessary tests? **Was not needed, the change that I made is very minor.**

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

**Review by LysandreJik — COMMENTED** [review]
LGTM, thanks!

**Review by sgugger — APPROVED** [review]
Yes this change is indeed needed! Thanks for tackling this!

File: src/transformers/data/data_collator.py [hunk 1]
```diff
@@ -446,7 +446,7 @@ def __call__(self, examples: List[Dict[str, torch.Tensor]]) -> Dict[str, torch.T
             "input_ids": input_ids,
             "attention_mask": self._tensorize_batch(attention_masks),
             "token_type_ids": self._tensorize_batch(segment_ids),
-            "masked_lm_labels": mlm_labels if self.mlm else None,
+            "labels": mlm_labels if self.mlm else None,
             "next_sentence_label": torch.tensor(nsp_labels),
         }
         if self.mlm:
```
Comment (vblagoje) [inline src/transformers/data/data_collator.py:452]: @gmihaila I also ran into this warning while doing BERT pretraining. I am wondering if we should remove lines 452 and 453 as they seem redundant.

Comment (gmihaila) [reply src/transformers/data/data_collator.py:452]: Yes you are perfectly right @vblagoje ! 452 and 453 are already coded inside of 449. I didn't look beyond that line 😄

Comment (vblagoje) [reply src/transformers/data/data_collator.py:452]: Ok, excellent @gmihaila . If you don't mind I can issue a PR. LMK

Comment (gmihaila) [reply src/transformers/data/data_collator.py:452]: I don't mind at all @vblagoje ! Thank you for noticing this! 😃
